ToDoList/Views/Shared/_NavBarPartial.cshtml

```diff
@@ -1,20 +1,20 @@
 ﻿@{
     <div class="bs-component">
         <nav class="navbar navbar-expand-lg navbar-light bg-light">
-            <a class="navbar-brand" href="#">²BᵉDᵒⁿᵉ</a>
+            <a class="navbar-brand" href="@Url.Action("Index", "Home")">²BᵉDᵒⁿᵉ</a>
             <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarColor03" aria-controls="navbarColor03" aria-expanded="false" aria-label="Toggle navigation">
                 <span class="navbar-toggler-icon"></span>
             </button>
             <div class="collapse navbar-collapse" id="navbarColor03">
                 <ul class="navbar-nav mr-auto">
                     <li class="nav-item active">
-                        <a class="nav-link" href="#">
+                        <a class="nav-link" href="@Url.Action("Index", "Home")">
```

Comment:
I forget what these razor syntax helper methods are called but good job making use of them here.